src/SFA.DAS.EAS.Web/Orchestrators/EmployerAccountTransactionsOrchestrator.cs

```diff
@@ -62,16 +62,22 @@ public EmployerAccountTransactionsOrchestrator(IMediator mediator, ICurrentDateT
                 transaction.PayeSchemeName = payeSchemeData?.PayeScheme?.Name ?? string.Empty;
             }
 
-            return new OrchestratorResponse<TransactionLineViewModel<LevyDeclarationTransactionLine>>
+            var outputData = new OrchestratorResponse<TransactionLineViewModel<LevyDeclarationTransactionLine>>
             {
                 Status = HttpStatusCode.OK,
                 Data = new TransactionLineViewModel<LevyDeclarationTransactionLine>
                 {
                     Amount = data.Total,
                     SubTransactions = data.Transactions,
-                    TransactionDate = data.Transactions.First().DateCreated
                 }
             };
+
+            if (data.Transactions.Any())
```

Comment:
This looks like `outputData.Data.TransactionDate` might not be set and so will default to `DateTime.MinValue`. Is that ok?

Reply:
Good spot, I have changed it to return a null data set if we have no declarations and reverted that block of code to the original. The acceptance test in turn now check that if we have a null dataset we expected zero levy and topup.

Reply:
Will returning a null view model break the views?

Reply:
No more than they already breaks! You should always be clicking through a "Parent" link to get to that screen, so should never find yourself there if there is no record. If you do fudge the Url to get there it current errors with sequence contains no element anyway!

I have tested in the UI and both payment and Levy drill downs continue to work when screen links used.